In [4]:
import pandas as pd
from matplotlib import pyplot as plt
import numpy as np

data = pd.read_csv('data_sets/train.csv')

(1306122, 3)

In [40]:
data[data.target == 1].sample(10)['question_text'].str.split()

902600     [How, can, I, find, more, information, about, ...
883251     [What, do, people, in, New, York, City, think,...
789169          [Why, do, people, lack, all, common, sense?]
1095516    [How/when, do, I, inform, someone, that, wants...
489300     [What, can, we, gather, from, the, fact, that,...
848402     [Did, ancient, Africans, just, sunbathe, or, d...
228350     [How, even, logically,, let, alone, legally,, ...
579809      [What, would, you, do, if, I, was, your, mummy?]
775803     [No, woman, likes, a, weak, man, and, most, me...
611796      [Is, Kerala, now, a, country, owned, by, goons?]
Name: question_text, dtype: object

data: 3 columns, 
1306122 observations, 808110 of which  insincere. => 6.2%
^^ model will learn to just predict sincere for all observations

insincere questions are on average 94.55 characters long; 16.36 words long
sincere questions are on average 65.43 characters long; 13.47 words long


